src/routes/api.py

```diff
@@ -1,12 +1,36 @@
-from fastapi import FastAPI, HTTPException
-from utils.logging import get_logger
+from fastapi import FastAPI
+
+from src.utils.logging import get_logger
+from src.utils.models import get_model
 
 logger = get_logger(__name__)
 
 app = FastAPI()
 
 
-@app.get("/chat")
-async def chat():
+def init_chat() -> dict:
```

Comment:
I am a bit concerned of putting the functions which contain the business logic into an API file. Maybe we can introduce the service layer and move them there? I don't have a strong opinion about it, just think, please, whether it would be a better option

Reply:
I think, we should use the async routes to be able to process multiple requests at the same time as  using async functions allows an API to handle concurrent requests more efficiently, especially for I/O-bound operations.

Agree, introducing service layer is good thing. 
It was just dummy routes.